src/main/webapp/app/assessment/assessment-header/assessment-header.component.html

```diff
@@ -144,7 +144,13 @@ <h3 class="top-container flex-wrap flex-lg-nowrap">
         <br />
         <div class="d-flex flex-wrap flex-lg-nowrap">
             <ng-container *ngIf="!result?.completionDate">
-                <button class="btn m-1 btn-primary" (click)="save.emit()" [disabled]="!assessmentsAreValid || !isAssessor || saveBusy || submitBusy || cancelBusy">
+                <button
+                    id="save"
+                    class="btn m-1 btn-primary"
+                    (click)="save.emit()"
+                    [disabled]="!assessmentsAreValid || !isAssessor || saveBusy || submitBusy || cancelBusy"
```

Comment:
Okay thanks! Yes but if someone changes the HTML, the client tests I added would fail, that's why I thought it should be okay, but your suggestion is better.👍